### DISCUSSION TITLE
ImportError: libtorch_cuda_cu.so: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by Caczhtus** [open-mmlab/mmdetection#9745] [Q&A] [answered]
``在 kaggle 云 GPU 环境上安装 mmdet ，发现到最后一步验证总报错

# 错误信息：
`/opt/conda/lib/python3.7/site-packages/mmcv/__init__.py:21: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  'On January 1, 2023, MMCV will release v2.0.0, in which it will remove '
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/tmp/ipykernel_23/2430390573.py in <module>
----> 1 from mmdet.apis import init_detector, inference_detector
      2 
      3 config_file = 'configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'
      4 # 从 model zoo 下载 checkpoint 并放在 `checkpoints/` 文件下
      5 # 网址为: [http://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth](http://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth%3C/span%3E)

/kaggle/working/mmdetection/mmdet/apis/__init__.py in <module>
      1 # Copyright (c) OpenMMLab. All rights reserved.
----> 2 from .inference import (async_inference_detector, inference_detector,
      3                         init_detector, show_result_pyplot)
      4 from .test import multi_gpu_test, single_gpu_test
      5 from .train import (get_root_logger, init_random_seed, set_random_seed,

/kaggle/working/mmdetection/mmdet/apis/inference.py in <module>
      6 import numpy as np
      7 import torch
----> 8 from mmcv.ops import RoIPool
      9 from mmcv.parallel import collate, scatter
     10 from mmcv.runner import load_checkpoint

/opt/conda/lib/python3.7/site-packages/mmcv/ops/__init__.py in <module>
      1 # Copyright (c) OpenMMLab. All rights reserved.
----> 2 from .active_rotated_filter import active_rotated_filter
      3 from .assign_score_withk import assign_score_withk
      4 from .ball_query import ball_query
      5 from .bbox import bbox_overlaps

/opt/conda/lib/python3.7/site-packages/mmcv/ops/active_rotated_filter.py in <module>
     10 ext_module = ext_loader.load_ext(
     11     '_ext',
---> 12     ['active_rotated_filter_forward', 'active_rotated_filter_backward'])
     13 
     14 

/opt/conda/lib/python3.7/site-packages/mmcv/utils/ext_loader.py in load_ext(name, funcs)
     11 
     12     def load_ext(name, funcs):
---> 13         ext = importlib.import_module('mmcv.' + name)
     14         for fun in funcs:
     15             assert hasattr(ext, fun), f'{fun} miss in module {name}'

/opt/conda/lib/python3.7/importlib/__init__.py in import_module(name, package)
    125                 break
    126             level += 1
--> 127     return _bootstrap._gcd_import(name[level:], package, level)
    128 
    129 

ImportError: libtorch_cuda_cu.so: cannot open shared object file: No such file or directory`

# 环境
`Package                               Version                Editable project location
------------------------------------- ---------------------- -------------------------
absl-py                               0.15.0
accelerate                            0.12.0
access                                1.1.8
addict                                2.4.0
affine                                2.4.0
aiobotocore                           2.4.2
aiohttp                               3.8.1
aioitertools                          0.11.0
aiosignal                             1.2.0
albumentations                        1.3.0
alembic                               1.9.2
allennlp                              2.10.1
altair                                4.2.2
annoy                                 1.17.1
ansiwrap                              0.8.4
anyio                                 3.6.1
apache-beam                           2.40.0
aplus                                 0.11.0
appdirs                               1.4.4
argon2-cffi                           21.3.0
argon2-cffi-bindings                  21.2.0
arrow                                 1.2.2
arviz                                 0.12.1
asn1crypto                            1.5.1
astroid                               2.13.4
astropy                               4.3.1
astunparse                            1.6.3
async-timeout                         4.0.2
asynctest                             0.13.0
atpublic                              2.3
attrs                                 21.4.0
audioread                             3.0.0
autocfg                               0.0.8
autopep8                              1.6.0
aws-requests-auth                     0.4.3
Babel                                 2.10.3
backcall                              0.2.0
backports.functools-lru-cache         1.6.4
backports.zoneinfo                    0.2.1
base58                                2.1.1
bayesian-optimization                 1.4.2
bayespy                               0.5.25
beatrix-jupyterlab                    3.1.7
beautifulsoup4                        4.11.1
bidict                                0.22.1
binaryornot                           0.4.4
biopython                             1.80
black                                 22.6.0
blake3                                0.2.1
bleach                                5.0.1
blinker                               1.4
blis                                  0.7.9
bokeh                                 2.4.3
Boruta                                0.3
boto3                                 1.26.60
botocore                              1.29.60
bq-helper                             0.4.1                  /src/bq-helper
bqplot                                0.12.36
branca                                0.6.0
brewer2mpl                            1.4.1
brotlipy                              0.7.0
cached-path                           1.1.6
cached-property                       1.5.2
cachetools                            4.2.4
Cartopy                               0.19.0.post1
catalogue                             2.0.8
catalyst                              22.4
catboost                              1.1.1
category-encoders                     2.6.0
certifi                               2022.12.7
cesium                                0.10.1
cffi                                  1.15.0
cftime                                1.6.2
chardet                               5.0.0
charset-normalizer                    2.1.0
chex                                  0.1.5
clang                                 5.0
cleverhans                            4.0.0
click                                 8.1.3
click-plugins                         1.1.1
cligj                                 0.7.2
cloudpickle                           2.1.0
cmaes                                 0.9.1
cmake                                 3.22.5
cmdstanpy                             1.1.0
cmudict                               1.0.13
colorama                              0.4.6
colorcet                              3.0.1
colorlog                              6.7.0
colorlover                            0.3.0
commonmark                            0.9.1
conda                                 22.9.0
conda-package-handling                1.8.1
confection                            0.0.4
configparser                          5.3.0
contextily                            1.2.0
contextlib2                           21.6.0
convertdate                           2.4.0
cookiecutter                          2.1.1
crcmod                                1.7
cryptography                          37.0.2
cudf                                  21.10.1
cufflinks                             0.17.3
cuml                                  21.10.2
cupy                                  9.6.0
cupy-cuda110                          11.5.0
CVXcanon                              0.1.2
cycler                                0.11.0
cymem                                 2.0.7
cysignals                             1.11.2
Cython                                0.29.33
cytoolz                               0.12.0
daal                                  2021.6.0
daal4py                               2021.6.3
dask                                  2022.2.0
dask-cudf                             21.10.1
dataclasses                           0.8
datasets                              2.1.0
datashader                            0.14.3
datashape                             0.5.2
datatable                             1.0.0
datatile                              1.0.3
deap                                  1.3.3
debugpy                               1.6.0
decorator                             5.1.1
defusedxml                            0.7.1
Delorean                              1.0.0
deprecat                              2.1.1
deprecation                           2.1.0
descartes                             1.1.0
dill                                  0.3.6
dipy                                  1.6.0
distlib                               0.3.6
distributed                           2022.2.0
dlib                                  19.24.0
dm-tree                               0.1.7
docker                                5.0.3
docker-pycreds                        0.4.0
docopt                                0.6.2
docstring-to-markdown                 0.11
docutils                              0.19
earthengine-api                       0.1.338
easydict                              1.10
easyocr                               1.6.2
ecos                                  2.0.12
eli5                                  0.13.0
emoji                                 2.2.0
en-core-web-lg                        3.3.0
en-core-web-sm                        3.3.0
entrypoints                           0.4
ephem                                 4.1.4
esda                                  2.4.3
essentia                              2.1b6.dev858
et-xmlfile                            1.1.0
etils                                 0.9.0
exceptiongroup                        1.1.0
explainable-ai-sdk                    1.3.3
explainers                            0.1
fairscale                             0.4.6
fastai                                2.7.10
fastapi                               0.89.1
fastavro                              1.6.1
fastcore                              1.5.27
fastdownload                          0.0.7
fasteners                             0.17.3
fastjsonschema                        2.15.3
fastprogress                          1.0.3
fastrlock                             0.8
fasttext                              0.9.2
fbpca                                 1.0
feather-format                        0.4.1
featuretools                          1.11.1
filelock                              3.7.1
Fiona                                 1.9.0
fitter                                1.5.2
flake8                                5.0.4
flashtext                             2.7
Flask                                 2.2.2
flatbuffers                           1.12
flax                                  0.6.4
flit_core                             3.7.1
folium                                0.14.0
fonttools                             4.33.3
frozendict                            2.3.4
frozenlist                            1.3.0
fsspec                                2023.1.0
funcy                                 1.18
fury                                  0.8.0
future                                0.18.2
fuzzywuzzy                            0.18.0
gast                                  0.4.0
gatspy                                0.3
gcsfs                                 2022.5.0
gensim                                4.0.1
geographiclib                         2.0
Geohash                               1.0
geojson                               3.0.0
geopandas                             0.10.2
geoplot                               0.5.1
geopy                                 2.3.0
geoviews                              1.9.6
ggplot                                0.11.5
giddy                                 2.3.3
gitdb                                 4.0.9
GitPython                             3.1.27
gluoncv                               0.10.5.post0
gluonnlp                              0.10.0
google-api-core                       1.31.5
google-api-python-client              2.52.0
google-apitools                       0.5.31
google-auth                           1.35.0
google-auth-httplib2                  0.1.0
google-auth-oauthlib                  0.4.6
google-cloud-aiplatform               0.6.0a1
google-cloud-appengine-logging        1.1.2
google-cloud-audit-log                0.2.2
google-cloud-automl                   1.0.1
google-cloud-bigquery                 2.2.0
google-cloud-bigtable                 2.10.1
google-cloud-core                     1.7.3
google-cloud-dataproc                 4.0.3
google-cloud-datastore                2.7.1
google-cloud-dlp                      3.7.1
google-cloud-firestore                2.5.3
google-cloud-kms                      2.11.2
google-cloud-language                 2.4.3
google-cloud-logging                  3.1.2
google-cloud-monitoring               2.9.2
google-cloud-pubsub                   2.13.0
google-cloud-pubsublite               1.4.2
google-cloud-recommendations-ai       0.2.0
google-cloud-resource-manager         1.5.1
google-cloud-scheduler                2.6.4
google-cloud-spanner                  3.15.1
google-cloud-speech                   2.14.1
google-cloud-storage                  1.44.0
google-cloud-tasks                    2.9.1
google-cloud-translate                3.7.4
google-cloud-videointelligence        2.7.1
google-cloud-vision                   2.7.3
google-crc32c                         1.1.2
google-pasta                          0.2.0
google-resumable-media                1.3.3
googleapis-common-protos              1.56.3
gplearn                               0.4.2
gpxpy                                 1.5.0
graphviz                              0.8.4
greenlet                              1.1.2
grpc-google-iam-v1                    0.12.4
grpcio                                1.51.1
grpcio-gcp                            0.2.2
grpcio-status                         1.47.0
gym                                   0.26.2
gym-notices                           0.0.8
h11                                   0.14.0
h2o                                   3.38.0.4
h5py                                  3.8.0
haversine                             2.7.0
hdfs                                  2.7.0
HeapDict                              1.0.1
hep-ml                                0.7.1
hijri-converter                       2.2.4
hmmlearn                              0.2.8
holidays                              0.19
holoviews                             1.15.4
hpsklearn                             0.1.0
html5lib                              1.1
htmlmin                               0.1.12
httplib2                              0.20.4
httptools                             0.5.0
huggingface-hub                       0.10.1
humanize                              4.5.0
hunspell                              0.5.5
husl                                  4.0.3
hydra-slayer                          0.4.0
hyperopt                              0.2.7
hypertools                            0.8.0
ibis-framework                        2.1.1
idna                                  3.3
igraph                                0.10.4
imagecodecs                           2021.11.20
ImageHash                             4.2.1
imageio                               2.19.3
imbalanced-learn                      0.10.1
imgaug                                0.4.0
implicit                              0.4.4
importlib-metadata                    6.0.0
importlib-resources                   5.10.2
inequality                            1.0.0
iniconfig                             2.0.0
ipydatawidgets                        4.3.2
ipykernel                             6.15.0
ipyleaflet                            0.17.2
ipympl                                0.7.0
ipython                               7.33.0
ipython-genutils                      0.2.0
ipython-sql                           0.3.9
ipyvolume                             0.6.0
ipyvue                                1.8.0
ipyvuetify                            1.8.4
ipywebrtc                             0.6.0
ipywidgets                            7.7.1
isort                                 5.11.5
isoweek                               1.3.3
itsdangerous                          2.1.2
Janome                                0.4.2
jax                                   0.3.25
jaxlib                                0.3.25+cuda11.cudnn805
jedi                                  0.18.1
jeepney                               0.8.0
jieba                                 0.42.1
Jinja2                                3.1.2
jinja2-time                           0.2.0
jmespath                              1.0.1
joblib                                1.0.1
json5                                 0.9.5
jsonlines                             1.2.0
jsonnet                               0.19.1
jsonschema                            4.6.1
jupyter                               1.0.0
jupyter-client                        7.3.4
jupyter-console                       6.4.4
jupyter-core                          4.10.0
jupyter-http-over-ws                  0.0.8
jupyter-lsp                           1.5.1
jupyter-server                        1.18.0
jupyter-server-mathjax                0.2.5
jupyter-server-proxy                  3.2.1
jupyterlab                            3.2.9
jupyterlab-git                        0.37.1
jupyterlab-lsp                        3.10.2
jupyterlab-pygments                   0.2.2
jupyterlab-server                     2.10.3
jupyterlab-widgets                    1.1.1
jupytext                              1.13.8
kaggle                                1.5.12
kaggle-environments                   1.12.0
keras                                 2.6.0
Keras-Preprocessing                   1.1.2
keras-tuner                           1.1.2
keyring                               23.6.0
keyrings.google-artifactregistry-auth 1.0.0
kiwisolver                            1.4.3
kmapper                               2.0.1
kmodes                                0.12.2
korean-lunar-calendar                 0.3.1
kornia                                0.5.8
kt-legacy                             1.0.4
kubernetes                            24.2.0
langcodes                             3.3.0
langid                                1.1.6
lazy-object-proxy                     1.9.0
learntools                            0.3.4
leven                                 1.0.4
Levenshtein                           0.20.9
libpysal                              4.7.0
librosa                               0.9.2
lightfm                               1.16
lightgbm                              3.3.2
lightning-utilities                   0.6.0.post0
lime                                  0.2.0.1
line-profiler                         4.0.2
llvmlite                              0.38.1
lmdb                                  1.4.0
lml                                   0.1.0
locket                                1.0.0
LunarCalendar                         0.0.9
lxml                                  4.9.2
Mako                                  1.2.4
mapclassify                           23.1
marisa-trie                           0.7.8
Markdown                              3.3.7
markdown-it-py                        2.1.0
markovify                             0.9.4
MarkupSafe                            2.1.2
matplotlib                            3.5.2
matplotlib-inline                     0.1.3
matplotlib-venn                       0.11.7
matrixprofile                         1.1.10
mccabe                                0.7.0
mdit-py-plugins                       0.3.0
mdurl                                 0.1.0
memory-profiler                       0.61.0
mercantile                            1.2.1
mgwr                                  2.1.2
missingno                             0.4.2
mistune                               0.8.4
mizani                                0.7.3
mlcrate                               0.2.0
mlens                                 0.2.3
mlxtend                               0.21.0
mmcv-full                             1.7.0
mmdet                                 2.28.1
mmh3                                  3.0.0
mne                                   1.3.0
mnist                                 0.2.2
mock                                  5.0.1
momepy                                0.5.4
more-itertools                        9.0.0
mpld3                                 0.5.9
mpmath                                1.2.1
msgpack                               1.0.4
msgpack-numpy                         0.4.8
multidict                             6.0.2
multimethod                           1.4
multipledispatch                      0.6.0
multiprocess                          0.70.14
munch                                 2.5.0
munkres                               1.1.4
murmurhash                            1.0.9
mxnet-cu110                           1.9.1
mypy-extensions                       0.4.3
nb-conda                              2.2.1
nb-conda-kernels                      2.3.1
nbclassic                             0.3.7
nbclient                              0.5.13
nbconvert                             6.4.5
nbdime                                3.1.1
nbformat                              5.4.0
nest-asyncio                          1.5.5
netCDF4                               1.6.2
networkx                              2.5
nibabel                               4.0.2
nilearn                               0.10.0
ninja                                 1.11.1
nltk                                  3.8.1
nnabla                                1.33.0
nnabla-ext-cuda110                    1.33.0
nose                                  1.3.7
notebook                              6.4.12
notebook-executor                     0.2
notebook-shim                         0.1.0
numba                                 0.55.2
numexpr                               2.8.4
numpy                                 1.21.6
nvtx                                  0.2.3
oauth2client                          4.1.3
oauthlib                              3.2.0
odfpy                                 1.4.1
olefile                               0.46
onnx                                  1.12.0
opencv-contrib-python                 4.5.4.60
opencv-python                         4.5.4.60
opencv-python-headless                4.5.4.60
openpyxl                              3.0.10
openslide-python                      1.2.0
opt-einsum                            3.3.0
optax                                 0.1.4
optuna                                3.1.0
orbax                                 0.1.0
orderedmultidict                      1.0.1
orjson                                3.7.5
ortools                               9.5.2237
osmnx                                 1.1.1
overrides                             6.1.0
packaging                             23.0
palettable                            3.3.0
pandarallel                           1.6.4
pandas                                1.3.5
pandas-datareader                     0.10.0
pandas-profiling                      3.1.0
pandas-summary                        0.2.0
pandasql                              0.7.3
pandocfilters                         1.5.0
panel                                 0.14.2
papermill                             2.3.4
param                                 1.12.3
parso                                 0.8.3
parsy                                 1.4.0
partd                                 1.3.0
path                                  16.6.0
path.py                               12.5.0
pathos                                0.3.0
pathspec                              0.9.0
pathtools                             0.1.2
pathy                                 0.10.1
patsy                                 0.5.2
pdf2image                             1.16.2
pexpect                               4.8.0
phik                                  0.12.2
pickleshare                           0.7.5
Pillow                                9.1.1
pip                                   22.1.2
platformdirs                          2.5.1
plotly                                5.13.0
plotly-express                        0.4.1
plotnine                              0.8.0
pluggy                                1.0.0
ply                                   3.11
pointpats                             2.2.0
polyglot                              16.7.4
pooch                                 1.6.0
portalocker                           2.7.0
pox                                   0.3.2
ppca                                  0.0.4
ppft                                  1.7.6.6
preprocessing                         0.1.13
preshed                               3.0.8
prettytable                           3.3.0
progressbar2                          4.2.0
prometheus-client                     0.14.1
promise                               2.3
prompt-toolkit                        3.0.30
pronouncing                           0.2.0
prophet                               1.1.1
proto-plus                            1.20.6
protobuf                              3.19.4
psutil                                5.9.1
ptyprocess                            0.7.0
pudb                                  2022.1.3
PuLP                                  2.7.0
py-lz4framed                          0.14.0
py-stringmatching                     0.4.2
py-stringsimjoin                      0.3.2
py4j                                  0.10.9.7
pyaml                                 21.10.1
PyArabic                              0.6.15
pyarrow                               5.0.0
pyasn1                                0.4.8
pyasn1-modules                        0.2.7
PyAstronomy                           0.18.0
pybind11                              2.10.3
pyclipper                             1.3.0.post4
pycocotools                           2.0.6
pycodestyle                           2.9.1
pycolmap                              0.3.0
pycosat                               0.6.3
pycparser                             2.21
pycrypto                              2.6.1
pyct                                  0.5.0
pycuda                                2022.1
pydantic                              1.8.2
pydegensac                            0.1.2
pydicom                               2.3.1
pydocstyle                            6.2.3
pydot                                 1.4.2
pydub                                 0.25.1
pyemd                                 0.5.1
pyerfa                                2.0.0.1
pyexcel-io                            0.6.6
pyexcel-ods                           0.6.0
pyfasttext                            0.4.6
pyflakes                              2.5.0
pygeos                                0.14
Pygments                              2.14.0
PyJWT                                 2.4.0
pykalman                              0.9.5
pyLDAvis                              3.2.2
pylint                                2.15.10
pymc3                                 3.11.5
PyMeeus                               0.5.12
pymongo                               3.12.3
Pympler                               1.0.1
pynndescent                           0.5.8
pynvml                                11.4.1
pynvrtc                               9.2
pyocr                                 0.8.3
pyOpenSSL                             22.0.0
pyparsing                             3.0.9
pypdf                                 3.3.0
PyPrind                               2.11.3
pyproj                                3.1.0
pyrsistent                            0.18.1
pysal                                 2.6.0
pyshp                                 2.3.1
PySocks                               1.7.1
pytesseract                           0.3.10
pytest                                7.2.1
python-bidi                           0.4.2
python-dateutil                       2.8.2
python-dotenv                         0.21.1
python-igraph                         0.10.4
python-Levenshtein                    0.20.9
python-louvain                        0.16
python-lsp-jsonrpc                    1.0.0
python-lsp-server                     1.7.1
python-slugify                        6.1.2
python-utils                          3.4.5
pythreejs                             2.4.1
pytoolconfig                          1.2.5
pytools                               2022.1.12
pytorch-ignite                        0.4.10
pytorch-lightning                     1.9.0
pytz                                  2022.1
pytz-deprecation-shim                 0.1.0.post0
pyu2f                                 0.1.5
PyUpSet                               0.1.1.post7
pyviz-comms                           2.2.1
PyWavelets                            1.3.0
PyYAML                                6.0
pyzmq                                 23.2.0
qgrid                                 1.3.1
qtconsole                             5.3.1
QtPy                                  2.1.0
quantecon                             0.6.0
quantities                            0.13.0
qudida                                0.0.4
quilt3                                5.1.1
randomgen                             1.23.1
rapidfuzz                             2.13.7
rasterio                              1.2.10
rasterstats                           0.17.0
ray                                   2.2.0
regex                                 2021.11.10
requests                              2.28.1
requests-futures                      1.0.0
requests-oauthlib                     1.3.1
resampy                               0.4.2
responses                             0.18.0
retrying                              1.3.3
rgf-python                            3.12.0
rich                                  12.6.0
rmm                                   21.10.1
rope                                  1.7.0
rsa                                   4.8
Rtree                                 1.0.1
ruamel-yaml-conda                     0.15.100
rvlib                                 0.0.6
s2sphere                              0.2.5
s3fs                                  2023.1.0
s3transfer                            0.6.0
sacremoses                            0.0.53
scattertext                           0.1.11
scikit-image                          0.19.3
scikit-learn                          1.0.2
scikit-learn-intelex                  2021.6.3
scikit-multilearn                     0.2.0
scikit-optimize                       0.9.0
scikit-plot                           0.3.7
scikit-surprise                       1.1.1
scipy                                 1.7.3
seaborn                               0.11.2
SecretStorage                         3.3.2
segregation                           2.3.1
semver                                2.13.0
Send2Trash                            1.8.0
sentencepiece                         0.1.97
sentry-sdk                            1.14.0
setproctitle                          1.3.2
setuptools                            59.8.0
setuptools-git                        1.2
shap                                  0.41.0
Shapely                               1.8.0
shortuuid                             1.0.11
simpervisor                           0.4
SimpleITK                             2.2.1
simplejson                            3.18.1
six                                   1.15.0
sklearn-contrib-py-earth              0.1.0+1.gdde5f89
sklearn-pandas                        2.2.0
slicer                                0.0.7
smart-open                            6.3.0
smhasher                              0.150.1
smmap                                 3.0.5
sniffio                               1.2.0
snowballstemmer                       2.2.0
snuggs                                1.4.7
sortedcontainers                      2.4.0
soundfile                             0.11.0
soupsieve                             2.3.1
spacy                                 3.3.2
spacy-legacy                          3.0.12
spacy-loggers                         1.0.4
spaghetti                             1.6.5
spectral                              0.23.1
spglm                                 1.0.8
sphinx-rtd-theme                      0.2.4
spint                                 1.0.7
splot                                 1.1.5.post1
spopt                                 0.4.1
spreg                                 1.3.0
spvcm                                 0.3.0
SQLAlchemy                            1.4.39
sqlparse                              0.4.2
squarify                              0.4.3
srsly                                 2.4.5
starlette                             0.22.0
statsmodels                           0.13.2
stemming                              1.0.1
stop-words                            2018.7.23
stopit                                1.1.2
stumpy                                1.11.1
sympy                                 1.10.1
tables                                3.7.0
tabulate                              0.9.0
tangled-up-in-unicode                 0.1.0
tbb                                   2021.8.0
tblib                                 1.7.0
tenacity                              8.0.1
tensorboard                           2.6.0
tensorboard-data-server               0.6.1
tensorboard-plugin-wit                1.8.1
tensorboardX                          2.5.1
tensorflow                            2.6.4
tensorflow-addons                     0.14.0
tensorflow-cloud                      0.1.14
tensorflow-datasets                   4.3.0
tensorflow-decision-forests           0.2.0
tensorflow-estimator                  2.6.0
tensorflow-gcs-config                 2.6.0
tensorflow-hub                        0.12.0
tensorflow-io                         0.21.0
tensorflow-metadata                   1.9.0
tensorflow-probability                0.14.1
tensorflow-serving-api                2.9.0
tensorflow-transform                  1.9.0
tensorpack                            0.11
tensorstore                           0.1.28
termcolor                             1.1.0
terminado                             0.15.0
terminaltables                        3.1.10
testpath                              0.6.0
text-unidecode                        1.3
textblob                              0.17.1
texttable                             1.6.7
textwrap3                             0.9.2
tfx-bsl                               1.9.0
Theano                                1.0.5
Theano-PyMC                           1.1.2
thinc                                 8.0.17
threadpoolctl                         3.1.0
tifffile                              2021.11.2
timm                                  0.6.12
tinycss2                              1.1.1
tobler                                0.9.0
tokenizers                            0.12.1
toml                                  0.10.2
tomli                                 2.0.1
tomlkit                               0.11.6
toolz                                 0.11.2
torch                                 1.11.0
torchaudio                            0.11.0
torchmetrics                          0.11.1
torchtext                             0.12.0
torchvision                           0.12.0
tornado                               6.1
TPOT                                  0.11.7
tqdm                                  4.64.0
traceml                               1.0.7
traitlets                             5.3.0
traittypes                            0.2.1
transformers                          4.20.1
treelite                              2.1.0
treelite-runtime                      2.1.0
trueskill                             0.4.5
tsfresh                               0.20.0
typed-ast                             1.5.4
typeguard                             2.13.3
typer                                 0.4.2
typing_extensions                     4.1.1
typing-utils                          0.1.0
tzdata                                2022.7
tzlocal                               4.2
ucx-py                                0.22.1+33.g9971bd1
ujson                                 5.3.0
umap-learn                            0.5.3
unicodedata2                          14.0.0
Unidecode                             1.3.4
update-checker                        0.18.0
uritemplate                           4.1.1
urllib3                               1.26.14
urwid                                 2.1.2
urwid-readline                        0.13
uvicorn                               0.20.0
uvloop                                0.17.0
vaex                                  4.16.0
vaex-astro                            0.9.3
vaex-core                             4.16.1
vaex-hdf5                             0.14.1
vaex-jupyter                          0.8.1
vaex-ml                               0.18.1
vaex-server                           0.8.1
vaex-viz                              0.5.4
vecstack                              0.4.0
virtualenv                            20.17.1
visions                               0.7.4
vowpalwabbit                          9.6.0
vtk                                   9.2.5
Wand                                  0.6.11
wandb                                 0.12.21
wasabi                                0.10.1
watchfiles                            0.18.1
wavio                                 0.0.7
wcwidth                               0.2.5
webencodings                          0.5.1
websocket-client                      1.3.3
websockets                            10.4
Werkzeug                              2.2.2
wfdb                                  4.1.0
whatthepatch                          1.0.4
wheel                                 0.37.1
widgetsnbextension                    3.6.1
witwidget                             1.8.0
woodwork                              0.16.4
Wordbatch                             1.4.9
wordcloud                             1.8.2.2
wordsegment                           1.3.1
wrapt                                 1.12.1
xarray                                0.20.2
xarray-einstats                       0.2.2
xgboost                               1.6.2
xvfbwrapper                           0.2.9
xxhash                                3.2.0
xyzservices                           2022.9.0
yacs                                  0.1.8
yapf                                  0.32.0
yarl                                  1.7.2
yellowbrick                           1.5
zict                                  2.2.0
zipp                                  3.8.0`

**Comment by RangeKing** [comment, marked as answer]
Please make sure your mmcv version matches Pytorch and CUDA version. Recommend to use `mim` to install `mmcv-full`.
```bash
pip install -U openmim
mim install mmcv-full
```
